sky/packages/sky/lib/sky_tool

```diff
@@ -29,6 +30,9 @@ APK_NAME = 'SkyShell.apk'
 ANDROID_PACKAGE = "org.domokit.sky.shell"
 ANDROID_COMPONENT = '%s/%s.SkyActivity' % (ANDROID_PACKAGE, ANDROID_PACKAGE)
 
+SKY_SHELL_APP_ID = 'com.google.SkyShell'
```

Comment:
That's from a change a couple of weeks ago to support code signing with google's keys.